source/manage/admin/generating-support-packet.rst

```diff
@@ -185,3 +185,12 @@ The Support Packet contains 3 go runtime profiling files:
 These files can be read using `pprof <https://golang.google.cn/cmd/pprof/>`_.
 
 Use ``go tool pprof -web X`` to open a visualization of the profile in your browser, replacing ``X`` with the profile's file name.
+
+Load metric
+------------
+
+From Mattermost v10.10, system admins have access to an additional diagnostic data point when working with Mattermost support by going to  the **Product** menu |product-list| and selecting **About Mattermost**. 
+
+The **Load Metric** field displays current server activity levels and usage patterns, giving Mattermost support teams a contextual reference point for understanding deployment characteristics without requiring access to detailed logs, configurations, or sensitive system data. System admins may need to reference this value when working with Mattermost support teams in environments where Support Packet generation is restricted or prohibited.
```

Comment:
I'm wondering if we shouldn't just be explicit that we're computing a monthly active users metric relative to the total number of licensed users?